featuremanagement/_models/_feature_flag.py

```diff
@@ -126,15 +126,15 @@ def telemetry(self):
         """
         return self._telemetry
 
-    def _validate(self):
+    def _validate(self) -> None:
         if not isinstance(self._id, str):
             raise ValueError(f"Invalid setting 'id' with value '{self._id}' for feature '{self._id}'.")
         if not isinstance(self._enabled, bool):
             raise ValueError(f"Invalid setting 'enabled' with value '{self._enabled}' for feature '{self._id}'.")
         self.conditions._validate(self._id)  # pylint: disable=protected-access
 
 
-def _convert_boolean_value(enabled):
+def _convert_boolean_value(enabled: Union[str, bool], feature_name: str) -> bool:
```

Comment:
Feels weird to pass in the name here only for the error message.

Reply:
This is so we can match the error message as we have in the other libraries. It also feels weird to do the check outside of this.